bundles/AdminBundle/src/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1784,25 +1806,25 @@ public function doBulkExportAction(Request $request): Response
         $result = [];
 
         foreach ($list as $item) {
-            if ($item['type'] == 'fieldcollection') {
+            if ($item['type'] === 'fieldcollection' && $this->getAdminUser()->isAllowed('fieldcollections')) {
```

Comment:
Like above, the admin user always has the permission

Reply:
? `$this->getAdminUser()` has nothing to do with admin users. It just returns the `\Pimcore\Model\User` object of the currently logged user - and this can or cannot have the permission `fieldcollections`.